src/js/game/LevelMVC/LevelView.js

```diff
@@ -1656,6 +1711,10 @@ module.exports = class LevelView {
     let sprite = null,
       frameList;
 
+    if (!LevelBlock.getMiniblockFrame(blockType)) {
+      return null;
+    }
+
```

Comment:
looks like this PR is actually made against a slightly old version of master; the latest version [already has an equivalent check](https://github.com/code-dot-org/craft/blob/master/src/js/game/LevelMVC/LevelView.js#L1714-L1717), making this redundant